downloads/index.md

```diff
@@ -267,23 +267,20 @@ Most Linux users should use the glibc binaries unless you know that your system
 
 Older releases of Julia for all platforms are available on the [Older releases page](/downloads/oldreleases/). Only the LTS and Stable releases are maintained.
 
-## "Nightly" builds
+## Nightly Builds
```

Comment:
This was quoted and the text expanded below because people thought they were actually built nightly

Reply:
I think we should explain the point in the text, since the quotes are a US-centric thing. It took me a few tries to guess what they were there for.

Reply:
Or perhaps we should not call them nightlies, but something like "Binaries sampled from the tip of master"?

Reply:
How about
```
## Nightly Builds (not actually nightly)
```
With that I think the existing text below it is ok?

Reply:
I added a sentence explaining nightlies.